controls/V-72001.rb

```diff
@@ -1,55 +1,80 @@
 # encoding: utf-8
 #
-=begin
------------------
-Benchmark: Red Hat Enterprise Linux 7 Security Technical Implementation Guide
-Status: Accepted
 
-This Security Technical Implementation Guide is published as a tool to improve
-the security of Department of Defense (DoD) information systems. The
-requirements are derived from the National Institute of Standards and
-Technology (NIST) 800-53 and related documents. Comments or proposed revisions
-to this document should be sent via email to the following address:
-disa.stig_spt@mail.mil.
-
-Release Date: 2017-03-08
-Version: 1
-Publisher: DISA
-Source: STIG.DOD.MIL
-uri: http://iase.disa.mil
------------------
-=end
-
-# Will need to be changed to reflect list of authorized system accounts
-system_accounts = attribute(
-  'system_accounts',
+KNOWN_SYSTEM_ACCOUNTS = attribute(
+  'known_system_accounts',
   default: [
-    'root',
-    'bin',
-    'daemon'
+        'root',
+        'bin',
+        'daemon',
+        'adm',
+        'lp',
+        'sync',
+        'shutdown',
+        'halt',
+        'mail',
+        'operator',
+        'nobody',
+        'systemd-bus-proxy',
+        'systemd-network',
+        'dbus',
+        'docker', # account used by the docker daemon
+        'polkitd',
+        'tss', #  Account used by the trousers package to sandbox the tcsd daemon
+        'postfix', # Service Account for Postfix Mail Daemon
+        'chrony', # Service Account for the Chrony Time Service
+        'sshd', # Service Account for SSH
+        'ec2-user', # Service Account for EC2 Access in AWS
```

Comment:
`ec2-user` should only be added if the target system is an AWS system lest someone stick this in as a 'backdoor' account.